### ISSUE TITLE
Is it possible to adjust the mpv config?

### ISSUE THREAD

**Issue by DomiStyle** [flathub/com.unicornsonlsd.finamp#17]
From what I have seen the internal player used is mpv?

Is it possible to adjust the config for the instance running inside of the Flatpak to enable replaygain until https://github.com/jmshrv/finamp/issues/765 is fixed?

I tried adding the config in `~/.var/app/com.unicornsonlsd.finamp/config/mpv/mpv.conf` but it doesn't seem like it is read.

**Comment by Chaphasilor** [comment]
Not really. It probably wouldn't work for transcoded streams (maybe not even streaming in general), since it relies on the tags inside the tracks. It would also potentially be inconsistent with Jellyfin (i.e. server having extracted a different LUFS value than what's in the tags).  

However, if there's a way to somehow pass a gain value to mpv, then we could think about adding proper support through just_audio_media_kit.

**Comment by DomiStyle** [comment]
> However, if there's a way to somehow pass a gain value to mpv, then we could think about adding proper support through just_audio_media_kit.

Yes, via the command line you can do:
```
mpv --replaygain=no --replaygain-fallback=-12.09 "test.flac"
```

Where `replaygain=no` is only needed so it takes the gain that is passed to it instead of using the metadata and -12.09 is the gain that is in the metadata or returned via the Jellyfin API.
